docs/howto-deploy-java-liberty-app.md

```diff
@@ -0,0 +1,235 @@
+Based on [Deploy a Java application with Open Liberty or WebSphere Liberty on an Azure Kubernetes Service (AKS) cluster](https://learn.microsoft.com/en-us/azure/aks/howto-deploy-java-liberty-app) at [8a7a0447](https://github.com/MicrosoftDocs/azure-docs-pr/commit/8a7a0447ed4a954a10bacdaf264d8affdce2f2ba).
+
+# Deploy a Java application with Open Liberty or WebSphere Liberty on an Azure Kubernetes Service (AKS) cluster
+
+This workshop demonstrates how to:
```

Comment:
This should mention how customer should use Liberty for AKS offer to create the cluster. Or it is mentioned somewhere else?

Reply:
I added this in 1dfb0ac.

> Workshop participants use an AKS cluster with Liberty pre-installed in advance. This has been done using the Azure Marketplace offer for Liberty on AKS. For a quick start guide on using the offer, see [Deploy a Java application with Open Liberty or WebSphere Liberty on an Azure Kubernetes Service (AKS) cluster](https://learn.microsoft.com/en-us/azure/aks/howto-deploy-java-liberty-app).